### ISSUE TITLE
Add Copy link to event to the Export menu

### ISSUE THREAD

**Issue by timgaunt** [datalust/seq-tickets#1436]
When diagnosing an issue we often find events of interest in the current filtered stream. 

At the moment to get a link to the event to share with your colleagues you have to find just that event and then copy the URL. 

The problem with this is you lose the context of the original filter which sometimes he is quite server intensive (especially when you are scrolling through a lot of events). 

It would be better if you could simply copy the current event URL in a similar way to the Perma link. 

Often these discussions are only for a few hours to a Perma Link is excessive

**Comment by nblumhardt** [comment]
Thanks for the feedback, Tim!

Strange coincidence - yesterday I went to copy a permalink and found myself (wrongly) dropping the Export menu to look for that option. This got me thinking that we should probably move permalinking under there.

(Permalinks don't actually cause the event to be retained, anymore - they just export its JSON to the metastore, and perform some merging trickery to make by-id queries for that event succeed even once it's been deleted from the underlying event store.)

How about if the options were:

 * Copy raw JSON
 * Download raw JSON
 * Copy direct link
 * Create permalink

?

**Comment by nblumhardt** [comment]
Hi Tim,

I've had a chance to think about this a little more; I've concluded that -

 * Because permalinks are no longer expensive, and
 * Because _Event > Find just this_ creates the non-permalinked version of the URL in the address bar

we probably shouldn't confuse things with an extra option, here.

I have condensed the Retain and Export menus, though, and since there's no longer mention of "retain"ing the event (when permalinks no longer have anything to do with retention), the whole experience should be a bit less confusing.

![image](https://user-images.githubusercontent.com/342712/155093494-6ff07cc6-b7a9-49fc-878c-be1ef7650e20.png)

I'll repurpose this ticket to track that change, unless you think I'm off down the wrong track :-)

Thanks again!
Nick
